js/popupView.js

```diff
@@ -0,0 +1,145 @@
+define(function(require) {
```

Comment:
could you switch to
```
define([
    'core/js/adapt'
], function (Adapt) {
```
Just to bring it in line with how we're doing it across the board now?